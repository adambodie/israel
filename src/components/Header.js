import React from 'react';
import { Link } from 'react-router-dom';
import FontAwesome from 'react-fontawesome';

const Header = () => (
				<header>
					<div className="top-bar">
						<div className="top-bar-left">
							<h4>Adam's Birthright Trip to The Land of Israel</h4>
							<Link to={'/index'}>
								<FontAwesome
									name='home'
									style={{ textShadow: '2px 2px 4px black' }}
									/>
							</Link>
						</div>
						<img src={'https://s3-us-west-2.amazonaws.com/birthright-israel.bodiewebdesign.com/images/main/header.jpg'} alt="RT-38-1047" />
					</div>
				</header>
);

export default Header;
