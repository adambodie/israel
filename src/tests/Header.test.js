import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow, mount, render } from 'enzyme';
import Header from '../components/Header';

it('renders without crashing', () => {
	const div = document.createElement('div');
	ReactDOM.render(<Header />, div);
	ReactDOM.unmountComponentAtNode(div);
});

it('renders correctly', () => {
	const tree = renderer.create(<Header />).toJSON();
	expect(tree).toMatchSnapshot();
});

it('renders a `header`', () => {
	const wrapper = shallow(<Header />);
    expect(wrapper.find('header').length).toEqual(1);
 });
