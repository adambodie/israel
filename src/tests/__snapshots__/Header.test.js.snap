// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header>
  <div
    className="top-bar"
  >
    <div
      className="top-bar-left"
    >
      <h4>
        Adam's Birthright Trip to Israel
      </h4>
    </div>
  </div>
</header>
`;
